plugins/dbnd-tensorflow/setup.py

```diff
@@ -14,7 +14,7 @@
 setuptools.setup(
     name="dbnd-tensorflow",
     package_dir={"": "src"},
-    install_requires=["dbnd==" + version, "tensorflow"],
+    install_requires=["dbnd==" + version, "tensorflow", "h5py==2.10.0"],
```

Comment:
we need it only in tox, we can not enforce the client to use a specific h5py library.

Reply:
what if your client has non compatible version?

Reply:
it's his problem to fix TensorFlow libraries issue

Reply:
ok